src/docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: '3.5'
+services: 
+  db: 
+    container_name: db
+    image: mcr.microsoft.com/mssql/server:2017-latest
+    environment:
+      SA_PASSWORD: "YourStrong!Passw0rd"
```

Comment:
I'm generally not a fan of keeping even default passwords in repos because people tend to not update them for a bit when getting started. We should have this be part of an initialization step within the RenameUtility console application